src/App.tsx

```diff
@@ -6,7 +6,7 @@ const Result = React.lazy(() => import("pages/Result"));
 
 const App: React.FC = () => {
     return (
-        <Suspense fallback={() => <p>Loading...</p>}>
```

Comment:
I think a more idiomatic solution would be to pass JSX in directly. Rather than creating a function, which is immediately run, and returns the same JSX.

e.g
```
<Suspense fallback={<p>Loading...</p>}>
...
```